resources/fetchersManager/test_helper.go

```diff
@@ -0,0 +1,131 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package fetchersManager
+
+import (
+	"context"
+	"github.com/elastic/cloudbeat/resources/fetchersManager/factory"
+	//"github.com/elastic/cloudbeat/resources/fetchersManager/registry"
+	"github.com/elastic/cloudbeat/resources/fetching"
+	"sync"
+)
+
+type NumberFetcher struct {
```

Comment:
Is this used?

Reply:
https://github.com/elastic/cloudbeat/blob/985b8d430992ba51084a7c21a247a13c23e25261/resources/fetchersManager/registry/registry_test.go#L78